packages/components/src/DataList/DataList.types.ts

```diff
@@ -51,9 +51,9 @@ export interface DataListSorting extends SortableOptions {
 }
 
 export interface SortableOptions {
-  readonly id: string;
-  readonly label: string;
-  readonly order: "asc" | "desc";
+  readonly id?: string;
+  readonly label?: string;
+  readonly order?: "asc" | "desc";
```

Comment:
you need all 3 no?

also, it's already optional 
![image](https://github.com/GetJobber/atlantis/assets/15986172/23dec93d-39e2-48c3-8923-49642c1cd748)

Reply:
But once I put it in JO, the setSort is telling me I'm missing `id` and `label`

Reply:
Resolving because we avoided this by adding the partial